openassessment/assessment/api/peer.py

```diff
@@ -45,7 +45,7 @@ def submitter_is_finished(submission_uuid, requirements):
         bool
 
     """
-    if not requirements:
+    if requirements is None:
```

Comment:
I could go either way. This was recently changed in our feature branch (Note master still uses `None` here https://github.com/edx/edx-ora2/blob/master/openassessment/assessment/api/peer.py#L48), to fix a bug. Then it popped up again in `student_training.py` here https://github.com/edx/edx-ora2/blob/master/openassessment/assessment/api/student_training.py#L44, which was the impetus for this PR to begin. As long as we're consistent between the locations I'm happy.

I just defaulted to what was here before we started doing any staff grading work, do you think we should change both locations to `if not requirements`, or leave it as `if requirements is None`? Changing it to the more general case *seems* to be okay, but I can't be 100% confident that the different handling of `None` and `{}` wasn't intended at some point.